client/components/TestRun/index.jsx

```diff
@@ -1390,6 +1409,7 @@ const TestRun = () => {
                         )}
                         patternName={testPlanVersion.title}
                         testerName={tester.username}
+                        // section:
```

Comment:
Would you be able to remove this comment and any other unrelated comments? 🙂

Reply:
Ok I just removed them